cgmanifest.json

```diff
@@ -1417,8 +1417,8 @@
         "type": "other",
         "other": {
           "name": "cal10n",
-          "version": "0.7.7",
-          "downloadUrl": "https://github.com/qos-ch/cal10n/archive/refs/tags/v_0.7.7.tar.gz"
+          "version": "0.8.1",
+          "downloadUrl": "https://cblmarinerstorage.blob.core.windows.net/sources/core/cal10n-0.8.1.tar.xz"
```

Comment:
How was this source generated? Can we link to the upstream source if it's available?

Reply:
It's borrowed from openSUSE's cal10n spec. Their source tarball has an extra folder that makes it work with their build.tar.xz file. The upstream source does not include this folder and would cause the build to fail. 
Is it ok for us to keep the current setup, or we would need to switch to a maven build instead?